dsgrid/config/project_config.py

```diff
@@ -49,7 +57,7 @@ class DimensionsModel(DSGBaseModel):
             " one dimension reference per type is allowed.",
         ),
     )
-    supplemental_dimensions: Optional[List[DimensionReferenceModel]] = Field(
+    supplemental_dimensions: List[DimensionReferenceModel] = Field(
```

Comment:
Many of our models unnecessarily define types with `Optional`. If a default value is provided then we don't want them declared `Optional`. That type means that it can be `None` or the real type.